ruby-sdk/Cargo.lock

```diff
@@ -335,31 +335,31 @@ dependencies = [
 
 [[package]]
 name = "eppo_client"
-version = "3.5.0"
+version = "3.5.1"
 dependencies = [
  "env_logger",
  "eppo_core",
  "log",
- "magnus 0.7.1",
+ "magnus",
```

Comment:
@rasendubi are we okay with unpinning these?

Reply:
It's not unpinning really. The actual version is still locked [below](https://github.com/Eppo-exp/eppo-multiplatform/pull/243/files#diff-ce3bd45b98c28ebeb3d63a78206eeb1969fff9143a4902e1541baf7c4baa3973R934)

I'm not sure of the exact specifics of the lock file format but it looks like it just had two version of magnus in it (0.7.1 and 0.6.4), so version was here to disambiguate. This proper lockfile updated removed 0.6.4, so there's more need for disambiguation. (That removal is expected as we just bumped magnus/serde_magnus in this release.)